policyengine_us/parameters/gov/states/nm/tax/income/deductions/medical_care_expense/head_of_household.yaml

```diff
@@ -0,0 +1,27 @@
+description: New Mexico provides the medical care expense deduction at these rates for individuals filing as head of household, depending on their adjusted gross income.
+metadata:
+  type: single_amount
+  threshold_unit: currency-USD
+  rate_unit: /1
+  label: New Mexico medical care expense deduction head of household filers rate
+  reference:
+    - title: 7-2-35. DEDUCTION - UNREIMBURSED OR UNCOMPENSATED MEDICAL CARE EXPENSES
+      href: https://klvg4oyd4j.execute-api.us-west-2.amazonaws.com/prod/PublicFiles/34821a9573ca43e7b06dfad20f5183fd/856ebf4b-3814-49dd-8631-ebe579d6a42b/Personal%20Income%20Tax.pdf
```

Comment:
could you link the pages to the pdfs

Reply:
all yaml files and references

Reply:
also please add [this](https://nmonesource.com/nmos/nmsa/en/item/4340/index.do#!fragment/zoupio-_Toc140503888/BQCwhgziBcwMYgK4DsDWszIQewE4BUBTADwBdoAvbRABwEtsBaAfX2zgEYAWABgFYeAZgAcogJQAaZNlKEIARUSFcAT2gBydRIiEwuBIuVrN23fpABlPKQBCagEoBRADKOAagEEAcgGFHE0jAAI2hSdjExIA) as a source